### PR TITLE
Improved UX of user logout

### LINKED ISSUE #5265: UX issues when a user logging out from an opened job
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
It suggest to keep unsaved changes
It sends accumulated logs and does not have any errors during logout

### Current Behaviour
It does not suggest keeping unsaved changes, logging out immediately
It tries to send logs to the server, but logout was first, so, saving logs is unsuccessful

### Possible Solution
Logout via a component

### Steps to Reproduce (for bugs)

1. Open a job
1. Create an object
1. Press logout
1.

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
Resolved #5265
Implemented by logging out via a component which saves logs
Also the component has dedicated URL ``/auth/logout`` and when a user goes there, the browser suggests to save unsaved changes on annotation view. 

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by bsekachev** [comment]
Currently failed because of npm connection error:
image

File: tests/cypress/integration/actions_users/issue_1810_login_logout.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ context('When clicking on the Logout button, get the user session closed.', () =
                     .trigger('mouseover', { which: 1 });
             });
             cy.get('span[aria-label="logout"]').click();
-            cy.url().should('include', `/auth/login/?next=/tasks/${taskId}`);
+            cy.url().should('include', `/auth/login?next=/tasks/${taskId}`);
```
Comment (klakhov) [inline tests/cypress/integration/actions_users/issue_1810_login_logout.js:49]: Header in this file needs to be updated

**Review by klakhov — COMMENTED** [review]
Nice patch, works as expected

**Review by klakhov — APPROVED** [review]

**Comment by bsekachev** [comment]
@kirill-sizov 

Do you have any ideas about failed REST API test on the server? I didn't touch the server here anyhow.

**Comment by kirill-sizov** [comment]
> @kirill-sizov
> 
> Do you have any ideas about failed REST API test on the server? I didn't touch the server here anyhow.

Never seen such an error, need some time to investigate

**Pull request merged**